examples/trusted_information_retrieval/client/rust/build.rs

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use std::path::Path;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let proto_path = Path::new("../../../../examples/trusted_information_retrieval/proto");
+    let file_path = proto_path.join("trusted_information_retrieval.proto");
+
+    // Tell cargo to rerun this build script if the proto file has changed.
+    // https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorerun-if-changedpath
+    println!("cargo:rerun-if-changed={}", file_path.display());
+
+    tonic_build::configure()
+        .build_client(true)
+        .build_server(false)
+        .compile(&[file_path.as_path()], &[proto_path])?;
+    Ok(())
```

Comment:
Is this something that we need to start encapsulating in a client library too?

Reply:
Looks like it

Reply:
Please create an issue at some point.

Reply:
Done https://github.com/project-oak/oak/issues/1120